app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -1679,37 +1679,45 @@ class YearToDateLiabilityController @Inject() (
                 }
 
               case _ =>
-                withTaxYear(fillingOutReturn.draftReturn) { taxYear =>
-                  commonDisplayBehaviour(
-                    nonCalculatedAnswers
-                  )(form =
-                    _.fold(
-                      _.taxDue.fold(nonCalculatedTaxDueForm)(a => nonCalculatedTaxDueForm.fill(a.inPounds())),
-                      c => nonCalculatedTaxDueForm.fill(c.taxDue.inPounds())
+                withResidentialStatus(fillingOutReturn.draftReturn) { wasUkResident =>
```

Comment:
do the unit tests consistently pass? Wherever you've added `withResidentialStatus` the residential status needs to be populated now in the corresponding tests